docs/_includes/ds6/listbox.html

```diff
@@ -0,0 +1,185 @@
+
+<div id="listbox">
+    <h2>@ebay/skin/listbox</h2>
+    <p>A listbox follows the principal of <a href="https://en.wikipedia.org/wiki/Progressive_disclosure">progressive disclosure</a>, it keeps secondary commands hidden from view until needed.</p>
```

Comment:
I would remove these paragraphs, and use the same text as we have for ds4 listbox.

"The skin listbox is intended for use as a custom view (or *facade*) for the native HTML select element. JavaScript will be required to create a 2-way binding between the two."